R/qenv-eval_code.R

```diff
@@ -68,6 +68,11 @@ setMethod("eval_code", signature = c("qenv", "expression"), function(object, cod
     object@warnings <- c(object@warnings, current_warnings)
     object@messages <- c(object@messages, current_messages)
   }
+
+  if (length(code) > 0) {
+    object@code_dependency[[length(object@code_dependency) + 1]] <- code_dependency(code, object@env)
+  }
```

Comment:
I think this line should be after `object@code <- c(object@code, code)`

Reply:
@gogonzo it was after `object@code <- c(object@code, code)` when it was evaluating the code inside `code_dependendency`. Now when `code_dependency` does not evaluate the code it's moved past the moment the code is evaluated

Reply:
Ahhh yes, I forgot about this. Should be fine 👍